exercises/practice/high-scores/.docs/instructions.append.md

```diff
@@ -0,0 +1,11 @@
+This exercise uses the [vector container][vector].
+You might find it useful to use the [algorithm][algorithm] header to solve this exercise.
+The last tests check for [immutability][immutability].
+None of the functions that you are tasked to write should change the scores of the game.
+
+If you want some extra practice, you can implement a function that adds new scores to the game.
+
+[vector] : https://en.cppreference.com/w/cpp/header/vector
```

Comment:
I'd rather link to the class not the header: https://en.cppreference.com/w/cpp/container/vector